ddtrace/contrib/psycopg/patch.py

```diff
@@ -61,17 +61,53 @@ def unpatch():
 class Psycopg2TracedCursor(dbapi.TracedCursor):
     """TracedCursor for psycopg2"""
 
-    def _trace_method(self, method, name, resource, extra_tags, *args, **kwargs):
```

Comment:
I can move this refactor to it's own PR. The issue I am trying to solve here is in pyscopg the query isn't always a string. The query can be a [Composable object](https://www.psycopg.org/docs/sql.html#psycopg2.sql.Composable). In pycopg we must convert the query to a string before calling `TracedCursor._trace_method()`

Reply:
Yes, move this to a separate PR.

Reply:
https://github.com/DataDog/dd-trace-py/pull/4449